docs/design/node_taint_eviction.md

```diff
@@ -42,51 +48,42 @@ automaticReplacementOptions:
   enableNodeFailureDetection: true
   taints:
   - key: "example.org/maintenance"
-    durationInSeconds: 7200 # Ensure the taint is present for at least 2 hours before replacing Pods on a node with this taint
-  - key: "*" # The wildcard would allow to define a catch all configuration
-    durationInSeconds: 3600 # Ensure the taint is present for at least 1 hour before replacing Pods on a node with this taint
+    durationInSeconds: 7200 # Mark Pods to be replaced after this taint key has been on the Node for at least 7200 seconds (i.e., 2 hours).
```

Comment:
What is the benefit here with those changes? Adding `taintReplacementTimeSeconds` makes sense, but I'm not sure about the value of adding yet another example?

Reply:
it's to show we can add a list of different taint keys and wildcard taint key.

Reply:
ok.